apps/condo/domains/organization/schema/InviteNewOrganizationEmployeeService.js

```diff
@@ -157,6 +157,8 @@ const InviteNewOrganizationEmployeeService = new GQLCustomSchema('InviteNewOrgan
                         user: {
                             id: user.id,
                         },
+                        phone,
```

Comment:
It looks like a wrong fix %) this is a message to USER not to phone/email.

Reply:
We have no other way by now to make it work :(